README.rst

```diff
@@ -31,6 +31,17 @@ The relevant specification documents used to write the `s1isp` software are:
    https://sentinels.copernicus.eu/documents/247904/2142675/Sentinel-1-SAR-Space-Packet-Protocol-Data-Unit.pdf
 
 
+Requirements and Installation
+-----------------------------
+
+The package requires Cython and the `bitstruct` module. Install them with the following commands::
+
+    pip install -r requirements.txt
+
+Build the package with::
+
+    make ext
+
```

Comment:
In principle the installations should be as easy as:
```
$ python3 -m pip install .
```
all the build and install requirements should be automatically managed by the `setuptools` and the `pyproject.toml` file, or
```
$ python3 -m pip install --editable .
```
for the editable mode.
The entire section that you propose is good but I would rename it into, e.g., "s1isp development" or something like that.
What do you think?